### PR TITLE
Create BLor11188 accord. to Strelcyns description

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient11188.xml [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">2+108+1</measure>
+                                    <measure unit="page" type="blank">5</measure><locus target="#1r #1v #Ir #Iv"/>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient11188.xml:187]: I count 4.

File: LondonBritishLibrary/orient/BLorient11188.xml [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+                                </msItem>
+                                <msItem xml:id="ms_i1.1.7">
+                                    <locus from="26v"/>
+                                    <title ref="LIT2509Weddas#Sunday"/>
+                                </msItem>
+                            </msItem>
+                            <msItem xml:id="ms_i1.2">
+                                <locus from="29v" to="42r"/>
+                                <title ref="LIT5646Mastagabe"/>
+                                <note>Sung on the <foreign xml:lang="gez">ʿǝzl</foreign> and the <foreign xml:lang="gez">gǝʿǝz</foreign> modes
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient11188.xml:73]: I think the keywords for the modes instead of the the element foreign could be used here.

**Review by thea-m — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient11188.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply LondonBritishLibrary/orient/BLorient11188.xml:73]: You mean, I shall add key to title? What is the correct key to be introduced for Geez and Ezl?

```
<msItem xml:id="ms_i1.2.1">
<title ref="LIT5646Mastagabe#Monday"/>
<msItem xml:id="ms_i1.2.1.1">
<locus from="29v"/>
<title key="ezl"/>
</msItem>
<msItem xml:id="ms_i1.2.1.2">
<locus from="30r"/>
<title key="geez"/></title>
</msItem>
</msItem>
```

Comment (thea-m) [reply LondonBritishLibrary/orient/BLorient11188.xml:73]: I meant  `<note>Sung in the <term key="EzelMode">ʿǝzl</term > and the <term key="GeezMode">gǝʿǝz</term > modes ...`
(https://betamasaheft.eu/Guidelines/?id=term)

Comment (CarstenHoffmannMarburg) [reply LondonBritishLibrary/orient/BLorient11188.xml:73]: I will change the title of the msItem only. The terms in the note need to stay like they are, correct?

**Review by DenisNosnitsin1970 — APPROVED** [review]

**Pull request merged**